client/src/containers/profile/file-upload/description/index.tsx

```diff
@@ -0,0 +1,70 @@
+import { FC } from "react";
+
+import { FileDownIcon } from "lucide-react";
+
+import { Button } from "@/components/ui/button";
+
+const downloadFiles = (files: iFile[]) => {
+  files.forEach((f) => {
+    const link = document.createElement("a");
+    link.href = f.path;
+    link.download = f.path.split("/").pop() || "";
+    document.body.appendChild(link);
+    link.click();
+    document.body.removeChild(link);
+  });
+};
```

Comment:
Wasn't it easier downloading a single `file.zip` containing the two files?

Reply:
Why? That would either mean we need a separate zip file, or create a zip on the fly, which adds more abstraction. Or I am missing something here?

Reply:
Yes, you would need an extra file containing both. I don't think this is big deal. About why? Well, first time I tried to download the files, the browser prompted me if I want to allow/deny the site downloading multiples at the same time, which is.. A little weird.

Also, you would save that loop above just pointing to the zip file. I prefer having an extra file than 9 additional lines of code, if you ask me.